### ISSUE TITLE
Incorrect case used for RawPrevResult CNIVersion check

### ISSUE THREAD

**Issue by anguslees** [containernetworking/cni#861]
https://github.com/containernetworking/cni/blob/5725f278690e2167982c25a08d1ff63247bb3753/pkg/version/version.go#L70-L75

I think the JSON field is `cniVersion` (different case)

**Comment by mars1024** [comment]
Good catch! Do you mind pulling  a request on this?

**Comment by anguslees** [comment]
Yep, will do.  I'm trying to write a test case, but I think #862 means it's always equivalent to ignoring the prevResult.cniVersion (which is also how this bug presents)?  What's a situation where the prevResult cniVersion matters?

**Comment by dcbw** [comment]
Yeah, testcase would be good here. Thought we had coverage for this, but clearly we don't, since I tried changing in my local tree and running tests, and nothing failed after I changed the case...
